### PR TITLE
feat: add zh-tw translation

### PR DESCRIPTION
**Pull request by EvanHsieh0415**

### CODE REVIEW

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+  "accessories.main_widget_position.toggle.disabled": [
+    "",
+    {"text": "[", "color": "gray"},
+    {"text": "→", "color": "#32a852"},
+    {"text": "]", "color": "gray"},
+    " 右"
+  ],
+  "accessories.main_widget_position.toggle.disabled.tooltip": "移動飾品欄按鈕到左側",
+
+  "accessories.group_filter.label": "群組過濾器：",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "accessories.group_filter.label": "群組篩選器：",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+  "accessories.group_filter.toggle.enabled": [
+    "",
+    {"text": "[", "color": "gray"},
+    {"text": "✔", "color": "#28FFBF"},
+    {"text": "]", "color": "gray"},
+    " 顯示"
+  ],
+  "accessories.group_filter.toggle.enabled.tooltip": "隱藏群組過濾器按鈕",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "accessories.group_filter.toggle.enabled.tooltip": "隱藏群組篩選器按鈕",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+  "accessories.group_filter.toggle.disabled": [
+    "",
+    {"text": "[", "color": "gray"},
+    {"text": "❌", "color": "#EB1D36"},
+    {"text": "]", "color": "gray"},
+    " 隱藏"
+  ],
+  "accessories.group_filter.toggle.disabled.tooltip": "顯示群組過濾器按鈕",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "accessories.group_filter.toggle.disabled.tooltip": "顯示群組篩選器按鈕",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+  ],
+
+  "accessories.unique_slots.toggle.show": "顯示獨特槽位",
+  "accessories.unique_slots.toggle.hide": "隱藏獨特槽位",
+  
+  "accessories.back.screen": "返回上次介面",
+
+  "accessories.open.screen": "開啟飾品介面",
+  
+  "accessories.reset.group_filter": "重設群組過濾器",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "accessories.reset.group_filter": "重設群組篩選器",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+  ],
+
+  "text.config.accessories.enum.screenType.none": "未決定",
+  "text.config.accessories.enum.screenType.original": "原始",
+  "text.config.accessories.enum.screenType.experimental_v1": "實驗性",
+
+  "text.config.accessories.option.screenOptions.showUnusedSlots": "顯示未使用的飾品欄",
+  "text.config.accessories.option.screenOptions.showUnusedSlots.tooltip": "切換顯示未使用的飾品欄",
+
+  "text.config.accessories.option.screenOptions.allowSlotScrolling": "允許欄位介面滾動",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "text.config.accessories.option.screenOptions.allowSlotScrolling": "允許欄位介面捲動",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 6]
```diff
@@ -0,0 +1,386 @@
...
+  "text.config.accessories.option.screenOptions.allowSlotScrolling.tooltip": [
+    "如果有模組會影響滾動時的物品操作，會在瀏覽飾品時出現問題，則可以關閉此選項以解決問題。"
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
    "如果有模組會影響捲動時的物品操作，會在瀏覽飾品時發生問題，則可以關閉此選項以解決問題。"
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 7]
```diff
@@ -0,0 +1,386 @@
...
+    {"text": "]", "color": "gray"},
+    " 分頁"
+  ],
+  "accessories.widget_type.paginated.tooltip": "有欄列的飾品布局",
+  "accessories.widget_type.scrollable": [
+    "",
+    {"text": "[", "color": "gray"},
+    {"text": "⏷", "color": "#953abd"},
+    {"text": "]", "color": "gray"},
+    " 滾動"
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
    " 捲動"
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 8]
```diff
@@ -0,0 +1,386 @@
...
+  ],
+  "accessories.widget_type.scrollable.tooltip": "可滾動的飾品布局",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "accessories.widget_type.scrollable.tooltip": "可捲動的飾品佈局",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 9]
```diff
@@ -0,0 +1,386 @@
...
+    {"text": "]", "color": "gray"},
+    " 啟用"
+  ],
+  "accessories.entity_look_at_cursor.toggle.enabled.tooltip": "禁止玩家模型看向游標",
+  "accessories.entity_look_at_cursor.toggle.disabled": [
+    "",
+    {"text": "[", "color": "gray"},
+    {"text": "❌", "color": "#EB1D36"},
+    {"text": "]", "color": "gray"},
+    " 禁用"
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
    " 停用"
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 10]
```diff
@@ -0,0 +1,386 @@
...
+    "Below is a set of screen options that you \n",
+    "can choose from depending on the style of \n",
+    "Modpack or your play style. \n\n",
+    "Do note that you can change such within the \n",
+    "Accessories Config if the given option is not \n",
+    "to your liking."
+  ],
+
+  "accessories.unique_slots.toggle.show": "顯示獨特槽位",
+  "accessories.unique_slots.toggle.hide": "隱藏獨特槽位",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "accessories.unique_slots.toggle.show": "顯示獨特欄位",
  "accessories.unique_slots.toggle.hide": "隱藏獨特欄位",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 11]
```diff
@@ -0,0 +1,386 @@
...
+  "#comment": "以下與設定相關",
+
+  "text.config.accessories.title": "Accessories 設定",
+
+  "text.config.accessories.option.useExperimentalCaching": "使用實驗性快取",
+  "text.config.accessories.option.useExperimentalCaching.tooltip": [
+    "嘗試快取各種物品堆疊查詢以提高效能，但可能會造成問題。"
+  ],
+
+  "text.config.accessories.category.clientOptions": "客戶端選項",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "text.config.accessories.category.clientOptions": "用戶端選項",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 12]
```diff
@@ -0,0 +1,386 @@
...
+  "text.config.accessories.category.clientOptions.tooltip": [
+    "包含用於渲染或玩家輸入的更通用的客戶端選項"
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
    "包含更多通用的用戶端選項，用於繪製或玩家輸入"
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 13]
```diff
@@ -0,0 +1,386 @@
...
+  ],
+
+  "text.config.accessories.option.clientOptions.forceNullRenderReplacement": "強制空渲染替換",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "text.config.accessories.option.clientOptions.forceNullRenderReplacement": "強制空繪製替換",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 14]
```diff
@@ -0,0 +1,386 @@
...
+  "text.config.accessories.option.clientOptions.forceNullRenderReplacement.tooltip": [
+    "強制任何已註冊的渲染，若發現沒有渲染，就改用預設渲染。"
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
    "強制任何已登錄的繪製，若發現沒有繪製，就改用預設繪製。"
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 15]
```diff
@@ -0,0 +1,386 @@
...
+  ],
+
+  "text.config.accessories.option.clientOptions.disableEmptySlotScreenError": "停用空飾品錯誤",
+  "text.config.accessories.option.clientOptions.disableEmptySlotScreenError.tooltip": [
+    "停用在打開飾品介面且未找到使用過的飾品欄時所顯示的錯誤訊息。"
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
    "停用在開啟飾品介面且未找到使用過的飾品欄時顯示的錯誤訊息。"
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 16]
```diff
@@ -0,0 +1,386 @@
...
+  ],
+
+  "text.config.accessories.option.clientOptions.showCosmeticAccessories": "顯示物品外觀渲染",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "text.config.accessories.option.clientOptions.showCosmeticAccessories": "顯示物品外觀繪製",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 17]
```diff
@@ -0,0 +1,386 @@
...
+  "text.config.accessories.option.clientOptions.showCosmeticAccessories.tooltip": [
+    "啟用/停用外觀物品變體（如果已裝備）的渲染。"
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
    "啟用/停用外觀物品變體（如果已裝備）的繪製。"
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 18]
```diff
@@ -0,0 +1,386 @@
...
+  ],
+
+  "text.config.accessories.option.clientOptions.disabledDefaultRenders": "停用預設渲染",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "text.config.accessories.option.clientOptions.disabledDefaultRenders": "停用預設繪製",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 19]
```diff
@@ -0,0 +1,386 @@
...
+  ],
+
+  "text.config.accessories.option.clientOptions.equipControl": "快速手動裝備模式",
+  "text.config.accessories.option.clientOptions.equipControl.tooltip": [
+    "變更從您的手快速裝備物品的模式。"
+  ],
+
+  "text.config.accessories.enum.playerEquipControl.must_crouch": "必須蹲下",
+  "text.config.accessories.enum.playerEquipControl.must_not_crouch": "不須蹲下",
+  "text.config.accessories.enum.playerEquipControl.disabled": "禁用",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "text.config.accessories.enum.playerEquipControl.disabled": "停用",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 20]
```diff
@@ -0,0 +1,386 @@
...
+  "text.config.accessories.option.screenOptions.showGroupTabs": "顯示快速尋找群組標籤",
+  "text.config.accessories.option.screenOptions.showGroupTabs.tooltip": [
+    "切換顯示或隱藏原始風格畫面的群組標籤。"
+  ],
+
+  "text.config.accessories.section.hover": "懸停效果",
+
+  "text.config.accessories.category.screenOptions.hoveredOptions": "懸停選項",
+  "text.config.accessories.category.screenOptions.hoveredOptions.tooltip": [
+    "當游標懸停在物品上時的渲染效果"
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
    "當游標懸停在物品上時的繪製效果"
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 21]
```diff
@@ -0,0 +1,386 @@
...
+  "text.config.accessories.category.screenOptions.unHoveredOptions.tooltip": [
+    "游標未懸停在物品上的渲染效果"
+  ],
+
+  "text.config.accessories.option.screenOptions.unHoveredOptions.renderUnHovered": "依然渲染未有游標懸停的物品",
+  "text.config.accessories.option.screenOptions.unHoveredOptions.darkenUnHovered": "降低物品亮度",
+  "text.config.accessories.option.screenOptions.unHoveredOptions.darkenedBrightness": "降低亮度",
+  "text.config.accessories.option.screenOptions.unHoveredOptions.darkenedOpacity": "降低不透明度",
+
+  "text.config.accessories.option.disabledDefaultRenders": "禁用預設渲染器",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "text.config.accessories.option.disabledDefaultRenders": "停用預設渲染器",
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 22]
```diff
@@ -0,0 +1,386 @@
...
+  "text.config.accessories.option.disabledDefaultRenders.tooltip": [
+    "停用指定飾品欄位的預設渲染，例如方塊或/和物品的渲染"
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
    "停用指定飾品欄位的預設繪製，例如方塊或/和物品的繪製"
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 23]
```diff
@@ -0,0 +1,386 @@
...
+  "accessories.slot.necklace": "項鍊",
+  "accessories.slot.back": "背部",
+  "accessories.slot.hand": "手部",
+  "accessories.slot.ring": "戒指",
+  "accessories.slot.wrist": "手腕",
+  "accessories.slot.belt": "腰帶",
+  "accessories.slot.anklet": "腳環",
+  "accessories.slot.shoes": "鞋子",
+  "accessories.slot.charm": "護符",
+  "accessories.slot.accessories.head": "頭盔",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "accessories.slot.accessories.head": "頭部",
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Comment by EvanHsieh0415** [comment]
review done

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 24]
```diff
@@ -0,0 +1,386 @@
...
+  ],
+
+  "text.config.accessories.option.clientOptions.showCosmeticAccessories": "顯示物品外觀繪製",
+  "text.config.accessories.option.clientOptions.showCosmeticAccessories.tooltip": [
+    "啟用/停用外觀物品變體（如果已裝備）的繪製。"
+  ],
+
+  "text.config.accessories.option.clientOptions.disabledDefaultRenders": "停用預設繪製",
+  "text.config.accessories.option.clientOptions.disabledDefaultRenders.tooltip": [
+    "如果正在使用，停用內建的預設渲染器渲染物品。"
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
    "如果正在使用，停用內建的預設渲染器繪製物品。"
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 25]
```diff
@@ -0,0 +1,386 @@
...
+  ],
+
+  "text.config.accessories.option.screenOptions.showEquippedStackSlotType": "顯示已裝備的飾品欄位類型",
+  "text.config.accessories.option.screenOptions.showEquippedStackSlotType.tooltip": [
+    "在物品提示中顯示已裝備的物品所在的飾品欄位類型。"
+  ],
+
+  "text.config.accessories.option.screenOptions.entityLooksAtMouseCursor": "使實體朝向游標",
+  "text.config.accessories.option.screenOptions.entityLooksAtMouseCursor.tooltip": [
+    "啟用時使背包中的實體模型看向滑鼠游標，反之當禁用時則允許手動旋轉實體模型。"
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
    "啟用時使背包中的實體模型看向滑鼠游標，反之當停用時則允許手動旋轉實體模型。"
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 26]
```diff
@@ -0,0 +1,386 @@
...
+  "text.config.accessories.option.screenOptions.hoveredOptions.brightenHovered": "提高亮度",
+  "text.config.accessories.option.screenOptions.hoveredOptions.cycleBrightness": "循環閃爍",
+
+  "text.config.accessories.option.screenOptions.hoveredOptions.line": "繪製線",
+
+  "text.config.accessories.option.screenOptions.hoveredOptions.clickbait": "點擊引導",
+
+  "text.config.accessories.category.screenOptions.unHoveredOptions": "未懸停選項",
+  "text.config.accessories.category.screenOptions.unHoveredOptions.tooltip": [
+    "游標未懸停在物品上的渲染效果"
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
    "游標未懸停在物品上的繪製效果"
```

File: common/src/main/resources/assets/accessories/lang/zh_tw.json [hunk 27]
```diff
@@ -0,0 +1,386 @@
...
+  ],
+
+  "text.config.accessories.option.screenOptions.unHoveredOptions.renderUnHovered": "依然渲染未有游標懸停的物品",
```
Comment (notlin4) [inline common/src/main/resources/assets/accessories/lang/zh_tw.json]:
```suggestion
  "text.config.accessories.option.screenOptions.unHoveredOptions.renderUnHovered": "依然繪製未有游標懸停的物品",
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Comment by Dragon-Seeker** [comment]
Is this PR ready to be pulled in or is there any more edits that need to be done?

**Comment by EvanHsieh0415** [comment]
> Is this PR ready to be pulled in or is there any more edits that need to be done?

Looks like the revisions are done and ready to be pulled.

**Pull request merged**